packages/ui/v2/core/Shell.tsx

```diff
@@ -238,7 +238,7 @@ function UserDropdown({ small }: { small?: boolean }) {
   }
   return (
     <Dropdown open={menuOpen}>
-      <DropdownMenuTrigger asChild onClick={() => setMenuOpen(true)}>
+      <DropdownMenuTrigger asChild onClick={() => setMenuOpen(!menuOpen)}>
```

Comment:
I think `setMenuOpen((menuOpen)=>!menuOpen)` is a logically better See https://stackoverflow.com/a/48394541/1109657

Reply:
@hariombalhara You're right, state changes like this should always be a function, thank you for reminding me! I'll make the change and push!